explorer/next-sitemap.config.mjs

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable sonarjs/no-duplicate-string -- comment */
+const URLS = {
+  "anvil-catalog-dev": "https://anvil-portal.dev.clevercanary.com",
```

Comment:
Instead of this, we can just take it from the environment directly and modify .env for each environment. 

LIke siteUrl: process.env.NEXT_PUBLIC_SITEMAP_DOMAIN]

Thanks!